github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -31,6 +31,35 @@ presubmits:
           resources:
             requests:
               memory: "8Gi"
+  - name: pull-cdi-unit-test-s390x
+    skip_branches:
+      - release-v\d+\.\d+
+    cluster: s390x-prow-virtual
+    annotations:
+      fork-per-release: "true"
+    always_run: true
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    labels:
+      preset-podman-in-container-enabled: "true"
+      preset-docker-mirror-proxy: "true"
```

Comment:
The image proxy is not available on the s390x cluster so this preset will have to be removed or the test pods will not be scheduled.

Reply:
Done